rootfs/scheduler/__init__.py

```diff
@@ -122,22 +122,10 @@ def deploy(self, namespace, name, image, command, **kwargs):  # noqa
             logger.info('No prior RC could be found for {}-{}'.format(namespace, app_type))
 
         # see if application or global deploy batches are defined
-        if not kwargs.get('batches', None):
-            # figure out how many nodes the application can go on
-            tags = kwargs.get('tags', {})
-            steps = len(self.get_nodes(labels=tags).json()['items'])
-        else:
-            steps = int(kwargs.get('batches'))
-
-        # figure out what kind of batches the deploy is done in - 1 in, 1 out or higher
-        if desired < steps:
-            # do it all in one go
-            batches = [desired]
-        else:
-            # figure out the stepped deploy count and then see if there is a leftover
-            batches = [steps for n in set(range(1, (desired + 1))) if n % steps == 0]
-            if desired - sum(batches) > 0:
-                batches.append(desired - sum(batches))
+        batches = kwargs.get('deploy_batches', None)
```

Comment:
Good catch - I separated this out from the Deployments PR so it changed to `deploy_batches`. Will update this PR to reflect that